hack/tools/Makefile

```diff
@@ -23,6 +23,7 @@ GOIMPORTS_VERSION=0.1.12
 VALE_VERSION=2.20.1
 GOLANGCI_LINT_VERSION=1.46.0
 MISSPELL_VERSION=0.3.4
+IMGPKG_VERSION=v0.11.0
```

Comment:
imgpkg actually has a version `0.35.0`.  Should we use it?

Reply:
+1 on using the latest version here.